backend/src/Service/CaptainAmountFromOrderCash/CaptainAmountFromOrderCashService.php

```diff
@@ -61,4 +67,16 @@ public function getCashOrdersPaymentsByCaptainId(int $captainId): array
     {
         return $this->captainAmountFromOrderCashManager->getCashOrdersPaymentsByCaptainId($captainId);
     }
+
+    // Is the captain allowed to edit?
+    public function getEditingByCaptain(int $orderId): ?bool
+    {
+        $captainAmountFromOrderCash = $this->captainAmountFromOrderCashManager->getCaptainAmountFromOrderCashByOrderId($orderId);
+     
+        if($captainAmountFromOrderCash === null) {
```

Comment:
Avoid using null in conditional statements, and as a set value.

Reply:
done